runtime/common/src/lib.rs

```diff
@@ -805,6 +805,34 @@ macro_rules! impl_config_traits {
                     ProxyType::Staking => matches!(c, RuntimeCall::ParachainStaking(..)),
                     #[cfg(not(feature = "parachain"))]
                     ProxyType::Staking => false,
+                    ProxyType::CreateMarket =>
+                        matches!(c,
+                            RuntimeCall::PredictionMarkets(zrml_prediction_markets::Call::create_market {..}) |
+                            RuntimeCall::PredictionMarkets(zrml_prediction_markets::Call::create_cpmm_market_and_deploy_assets {..})
```

Comment:
I think this call provides too many privileges, as it also invokes `buy_complete_set`

Reply:
Yeah, this call would require liquidity and market creation privileges. I think it's fine to just remove `create_cpmm_market_*` completely because CPMM is a done deal anyways and it's for permissionless markets only.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1038/commits/b80ae75c2da1bc7ce687cb5e2a5aa6d1aef2d37c